entry_types/scrolled/package/src/frontend/Video.js

```diff
@@ -1,103 +1,19 @@
-import React, {useRef, useEffect, useContext} from 'react';
-import classNames from 'classnames';
-
-import ScrollToSectionContext from './ScrollToSectionContext';
-import MutedContext from './MutedContext';
-
-import styles from './Video.module.css';
-
+import React from 'react';
+import {useFile} from '../entryState';
+import {VideoPlayer} from './VideoPlayer';
+
+/**
+ * Render a video file.
+ *
+ * @param {Object} props
+ * @param {number} props.id - Perma id of the video file.
+ */
```

Comment:
Needs to be added to `documentation.yml` to make it appear at the right position in the TOC of the API doc.